src/templates/mht_part.tpl

```diff
@@ -0,0 +1,6 @@
+------=_NextPart_000_0000_01D0CFF2.897FE540
```

Comment:
Do you have any idea what it means? Yes, I know, that it is reverse-engineered :)